src/ui/components/atoms/side-menu-button-site-seo/SideMenuButtonSiteSeo.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import classNames from 'classnames';
+import { Icon } from '../../utils/svg-icons/icons';
+import type { IProps } from './types/IProps';
+import styles from './styles.module.scss';
+
+function SideMenuButtonSiteSeo({
+  children,
+  icon,
+  color,
+  variant = 'primary',
+  path,
+  basePath,
+}: IProps) {
+  const [isActive, setIsActive] = useState(false);
+  const handleClick = () => {
+    setIsActive(!isActive);
+    if (path) {
+      window.location.href = path;
```

Comment:
remove this line of code it is not necessary, and pass the hadleClick from props